<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_tremA">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(239, 41, 41)}</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2AB">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4ABD">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3AB">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1A">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ligar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3BC">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2BAC">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4BADEC">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_tremB">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(52, 101, 164)}</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2CB">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3C">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4CBE">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1DA">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4DE">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3DBE">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3EC">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4ED">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(186, 189, 182)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_tremC">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(115, 210, 22)}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_tremD">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(117, 80, 123)}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_tremE">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(252, 233, 79)}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_B">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>292</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {background: rgb(52, 101, 164)}</string>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_C">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>292</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {background: rgb(115, 210, 22)}</string>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_E">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>292</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {background: rgb(252, 233, 79)}</string>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_D">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>292</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {background: rgb(117, 80, 123)}</string>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_A">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>292</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {background: rgb(239, 41, 41)}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_A">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>332</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_B">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>362</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_C">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>392</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_D">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>422</y>
      <width>20</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_E">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>9</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <zorder>label_trilho1DA</zorder>
   <zorder>label_trilho3DBE</zorder>
   <zorder>label_trilho2AB</zorder>
   <zorder>label_trilho4ABD</zorder>
   <zorder>label_trilho3AB</zorder>
   <zorder>label_trilho1A</zorder>
   <zorder>label_tremA</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_trilho3BC</zorder>
   <zorder>label_trilho2BAC</zorder>
   <zorder>label_trilho4BADEC</zorder>
   <zorder>label_tremB</zorder>
   <zorder>label_trilho2CB</zorder>
   <zorder>label_trilho3C</zorder>
   <zorder>label_trilho4CBE</zorder>
   <zorder>label_trilho4DE</zorder>
   <zorder>label_trilho3EC</zorder>
   <zorder>label_trilho4ED</zorder>
   <zorder>label_tremC</zorder>
   <zorder>label_tremD</zorder>
   <zorder>label_tremE</zorder>
   <zorder>horizontalSlider_B</zorder>
   <zorder>horizontalSlider_C</zorder>
   <zorder>horizontalSlider_E</zorder>
   <zorder>horizontalSlider_D</zorder>
   <zorder>horizontalSlider_A</zorder>
   <zorder>label_A</zorder>
   <zorder>label_B</zorder>
   <zorder>label_C</zorder>
   <zorder>label_D</zorder>
   <zorder>label_E</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
